source/creating_content_with_studio/configuring_blocks/add-chart.rst

```diff
@@ -0,0 +1,86 @@
+Adding a chart
+==============
+
+You can use charts to create a graphic representation of your data.
+
+Step 1: Select the block type
+-----------------------------
+
+You need to define the block type before you configure the content you want to add.
+
+1. From the preview pane, click the block you want to edit.
+2. From the left side pane, select **Chart** as the block type.
+3. Click **Next**.
+
+If you change your mind, you can change the block type afterward.
+The block configuration will be reset after you change the block type.
+
+Step 2: Select the dataset
+--------------------------
+
+Select the source dataset to create a graphic representation of the data.
+
+.. admonition:: Note
+    :class: note
+ 
+    The dataset is only selected for the current block.
+    If you add some content to another block afterward, you will need to select an input dataset again.
+
+1. From the left side pane, select the desired dataset from the list.
+
+   A preview of the first 20 records opens in the preview pane.
+
+   <PLACEHOLDER_SCREENSHOT_DATASET_RECORDS_PREVIEW>
+
+2. Click **Next** to continue with the configuration.
+
+Step 3: Select the visualization type
+-------------------------------------
+
+After selecting a block type and the input dataset, you can now select the type of chart to be inserted into the block.
+
+1. From the left side pane, select the desired type of chart.
+2. Click **Next** to continue with the configuration.
+
+Step 4: Configure the visualization parameters
+----------------------------------------------
+
+1. From the left side pane, configure the axes of the chart:
+
+   a. Select the dataset field to be used on the X-axis (aka category or horizontal axis). This axis is generally used to categorize data. When using a date field, select the time scale to be used.
+   b. Select the function to apply and the field to be used on the Y-axis (aka values or vertical axis). This axis is generally used to measure data.
+
+2. (optional) You can also refine the data shown on the chart:
+
+   a. Under Y-axis, click **Add a condition**.
```

Comment:
It's just a suggestion but since the sections header can differ here, we might want to rephrase it: "Under the second section (values or vertical axis), click ..."